external-api/src/createApp.ts

```diff
@@ -1,4 +1,3 @@
-/* eslint-disable @typescript-eslint/no-explicit-any */
 import { generateApp } from './generateApp';
 
-export const app: any = generateApp();
+export const app = generateApp();
```

Comment:
I'm happy to, but TypeScript is already able to accurately infer the type as `Express` (because `generateApp` returns the result of calling the `express()` function from the `express` package`).